src/libslic3r/PrintConfig.hpp

```diff
@@ -1067,6 +1067,9 @@ PRINT_CONFIG_CLASS_DEFINE(
     ((ConfigOptionBool,                enable_filament_ramming))
     ((ConfigOptionBool,                support_multi_bed_types))
 
+    // Small Area Infill Flow Compensation
+    ((ConfigOptionBool,                 small_area_infill_flow_compensation))
```

Comment:
Can we make a `PrintRegionConfig` member so that it would make AB test easier as it allows us to print two objects or modifiers at once, one with compensation and another without?

Reply:
Done